pyproject.toml

```
@@ -114,9 +114,9 @@ dependencies = [
 ]
 
 [tool.hatch.envs.docs.scripts]
-build = "mike deploy --push dev"
-serve = "mike serve"
-release = "version=$(python -W ignore -c 'import dagfactory; print(dagfactory.__version__)') && mike deploy --push --update-aliases $version latest"
+dev = "mkdocs build && mkdocs serve" # For local development and preventing publishing
+gh_deploy = "mike deploy --push dev"
+gh_release = "version=$(python -W ignore -c 'import dagfactory; print(dagfactory.__version__)') && mike deploy --push --update-aliases $version latest"
```

```suggestion
gh-deploy = "mike deploy --push dev"
gh-release = "version=$(python -W ignore -c 'import dagfactory; print(dagfactory.__version__)') && mike deploy --push --update-aliases $version latest"
```
Can we use `-` separators. I guess they suit well for commands similar to how we have for `test-cov`, `test-integration`, `static-check`, etc defined earlier in this file.